community.html

```diff
@@ -0,0 +1,77 @@
+---
+title: "Community Members"
+layout: default
+comment: >-
+  Links to people involved with the CHiMaD phase field
```

Comment:
Introductory text should set the scope and explain who gets in the list, and who does not. For example,
> The CHiMaD Phase Field working group is a growing community led by respected leaders in the field with valuable contributions from an international group of graduate students, postdocs, and researchers. This gallery provides biographical sketches and contact information for the active participants.